### ISSUE TITLE
Transformation verification

### ISSUE THREAD

**Issue by bmiles** [dacarlin/robot-bagel#1]
> Transformation, growth (or grow from Kunkel glycerol stocks, workflow would be Kunkel, transform, selection plate, mail selection plate to Genscript, rec sequencing within 48 hours, grow appropriate clone)

We can do a miniprep of transformants after growth and do Sanger seq on Transcriptic for this.

> Grow in triplicate (abstract away the idea of plates/wells?)

**Comment by dacarlin** [comment]
Yes, we could, but that adds significant cost. At $5 per miniprep and 2 x $7.50 = $15 per sequencing run, for three clones, is $60. Plus about $20 per mutant in reagents and workcell time for the Kunkel, and we're looking at $80 per mutant

**Comment by dacarlin** [comment]
That said, I am happy to write it using Transcriptic mini prep and sequencing for now!
